NewFile.txt

```diff
@@ -0,0 +1,7 @@
+This is a new file.
+
+With some new content.
+
+Seriously...
+
+Very important content.
```

Comment:
Reply

---
In reply to: [70324401](https://github.com/kivancmuslu/GitHubTest/pull/1#discussion_r70324401) [](ancestors = 70324401)